text/0000-project-groups.md

```diff
@@ -0,0 +1,385 @@
+
+8. Archive infrastructure.
+
+- Archive GitHub repository to be read-only.
+- Archive chat channel(s) on any platforms.
+
+9. (Optional) Restore group
+
+- At any later point the group could be restored to active status if there are
+  assigned liaisons and shepherds, and the group has consensus from the team
+  that the group should become active again.
+- If significant time has passed, part of restoring the group should be to
+  evaluate whether the past decisions and rationale are still applicable to the
+  present.
+- If there is consensus to become active again, go to step 3.
+
+[`rust-lang/team`]: https://github.com/rust-lang/team
+[internals]: https://internals.rust-lang.org
+
+# Reference-level explanation
+
+## Common Aspects of Working Groups and Project Groups
+
+Before going into the differences between these groups it's important to start
+with what it is shared between them.
+
+- A charter that defines the scope and intent of the group.
+
+- A GitHub repository hosted under the `rust-lang` organization
+  (or separate organisation if necessary) containing the charter and
+  instructions for how community members can monitor or participate in
+  the group.
+
+- Groups have at least one shepherd who acts as the leader of the group and is
+  typically responsible for writing the initial charter, handling administrative
+  and communication tasks, as well as delegating responsibilities to other
+  members in the group.
+
+- Groups should have a liaison member associated with an official Rust team.
+
+  - This liaison may also be but is not required to be a shepherd.
+
+- Groups and their members are represented on the official
+  [rust-lang.org] website.
+
+- Membership requirements for both groups is decided by the shepherd and should
+  be stated in the charter.
+
+  - Initial membership should try to represent people who have already been
+    participating regularly and productively in the respective area.
+
+- Neither group has _"formal decision making power"_: meaning that they are not
+  able to accept RFCs on `rust-lang/rfcs`. Similarly, neither group has
+  representation on the Core team.
+
+  - Groups are of course encouraged to create RFCs as well
+    as advocate their concerns and desired changes to the Rust teams
+    and community.
+
+- Both groups are entitled to spaces in any or all of Rust's officially managed
+  discussion forums.
+
+  - As the time of this writing this includes [Zulip] and [Discord].
+  - Groups are also free to create spaces on platforms where the Rust project
+    does not have a official presence. Moderation for these spaces should still
+    follow Rust's [Code of Conduct].
+
+[rust-lang.org]: https://rust-lang.org
+[zulip]: https://rust-lang.zulipchat.com
+[discord]: https://discord.gg/rust-lang
+[code of conduct]: https://www.rust-lang.org/policies/code-of-conduct
+
+### Initial Setup
+
+Once a group has been approved, a pull request with the initial set of members
```

Comment:
Who does a group have to be approved by? Just a general RFC consensus or would some team be responsible for gate-keeping new groups?

Reply:
As stated in the RFC, I interpret this as the relevant team(s) deciding themselves how groups under them are approved. For example, it might be an RFC like https://github.com/rust-lang/rfcs/pull/2797 which is FCPed, or it might be more light-weight.

Reply:
* [Working Group Creation](https://github.com/XAMPPRocky/rfcs/blob/project-groups/text/0000-project-groups.md#working-group-creation)
* [Project Group Creation](https://github.com/XAMPPRocky/rfcs/blob/project-groups/text/0000-project-groups.md#project-group-creation)
TL;DR is that working groups are approved by core, and project groups are approved by their relevant teams. See ["Future Work"](https://github.com/XAMPPRocky/rfcs/blob/project-groups/text/0000-project-groups.md#future-work) for how we'd want to achieve consensus.